### ISSUE TITLE
installation instructions

### ISSUE THREAD

**Issue by Sreevatsa123** [jprante/elasticsearch-jdbc#758]
Sir,the following 2 points mentioned under Installation couldn't be understood by me:
1)modify script in the bin directory to your needs (Elasticsearch cluster address)

2)run script with a command that starts org.xbib.tools.JDBCImporter with the lib directory on the classpath.
Could you please elaborate on them?

**Comment by jprante** [comment]
Can you elaborate what you not understand? The instructions are very clear.

**Comment by Sreevatsa123** [comment]
Yes sir.The instruction are very very clear.I dint understand how to a run a script that starts org.xbib.tools.JDBCImporter with the lib directory on the classpath.
(PS:I am sorry..i am a beginner..So i find difficult to catch up with a few things)

**Comment by jprante** [comment]
You know how to use the command line?

It's similar to starting Elasticsearch.

**Comment by Sreevatsa123** [comment]
yes sir i know to us CL.what should i do to put lib on the classpath?There are many scripts.Which to run?(I am sorry if i am sounding dumb,but i am really facing difficulties)

**Comment by jprante** [comment]
You can experiment with the scripts and modify them as you like. Just as the instructions tell you.
